trunk/SUAVE/Attributes/Solids/BiCF.py

```diff
@@ -0,0 +1,31 @@
+## @ingroup Attributes-Solids
+
+# B[a-z]_[a_z]F.py
+#
+# Created: Jul 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from Solid import solid
+from SUAVE.Core import Data, Units
+
+#-------------------------------------------------------------------------------
+# Bi-Directional Carbon Fiber Solid Class
+#-------------------------------------------------------------------------------
+
+class bidirectional_carbon_fiber(solid):
+
+    """ Physical Constants of a Specific to Bi-Directional Carbon Fiber"""
+
+    def __defaults__(self):
+
+        self.ultimate_tensile_strength        = 600e6    *Units.Pa
```

Comment:
can you add periods to all of these?

Reply:
Usage of exponential notation also forces float data type.